### PR TITLE
Component: Logo PR

### PR DESCRIPTION
**Pull request by torykling**
#15 

I built a logo component with a size property to make it smaller for nav and a clickable property to add a link to return to the home page. 

Question: Do the sizes seem right? I set the default size to be 20vh by 20vh and the small to be 10vh by 10vh.

### CODE REVIEW

File: client/src/components/Logo/Logo.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import "./Logo.css";
+
+// Function based React Component
+const Logo = props => {
+  // Default Class to apply to Component
+  let classList = `Logo`;
+  if (props.small) {
+    classList += ` smallLogo`;
```
Comment (ZakkMan) [inline client/src/components/Logo/Logo.js]: Can you change this class name to `small-logo`? We generally want our classes to be kebab case

File: client/src/components/Logo/Logo.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  }
+  if (props.clickable) {
+    return (
+      <div className={classList}>
+        <Link to="/" />
+      </div>
+    );
+  } else {
+    return <div className={classList}></div>;
+  }
```
Comment (ZakkMan) [inline client/src/components/Logo/Logo.js]: This is awesome! Can you make it so someone can pass in the destination URL as a prop, with the default value as `"/"`? We'll almost always want the logo to link to the homepage (if it is clickable) but there may be cases where we want to change that.

Comment (ZakkMan) [reply client/src/components/Logo/Logo.js]: Let's call the prop `to`, to keep it consistent with React Router

File: client/src/components/Logo/Logo.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
+.Logo {
+  background-image: url("./logo.png");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 20vh;
+  height: 20vh;
```
Comment (ZakkMan) [inline client/src/components/Logo/Logo.css]: I would do this in pixels or rems - maybe like `3rem` or `4rem` for large and `2rem` for small?

**Comment by torykling** [comment]
@ZakkMan Thanks for reviewing this! I made some changes. Do you think these are okay? 

Also am I supposed to fill out a new pull request somehow to resubmit for review or should I just write a comment like this?

**Pull request merged**